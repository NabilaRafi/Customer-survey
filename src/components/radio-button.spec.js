import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import RadioButton from './radio-button';

describe('RadioButton', () => {
    const outputSnapshot = shallow(<RadioButton />);
    it('Render sectionPanel without crashing', () => {        
        expect(shallowToJson(outputSnapshot)).toMatchSnapshot();
    })
    it('should render label component', () => {
        expect(outputSnapshot.find('label').length).toEqual(2);
    })
    it('should render input component with type as radio', () => {
        expect(outputSnapshot.find('input').length).toEqual(2);
    })
})