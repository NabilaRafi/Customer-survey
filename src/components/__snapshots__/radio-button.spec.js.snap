// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton Render sectionPanel without crashing 1`] = `
<div>
  <label>
    <input
      checked={true}
      onChange={[Function]}
      type="radio"
      value="yes"
    />
    yes
  </label>
  <label>
    <input
      checked={false}
      onChange={[Function]}
      type="radio"
      value="no"
    />
    no
  </label>
</div>
`;
